demos/ruby-hello/snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: "ruby-hello"
+version: 0.1
+summary: Test ruby
+description: |
+  Snap to test ruby compiled correctly, exists, and works
+
+grade: stable
+confinement: strict
+
+apps:
+  ruby-hello:
+    command: ruby-hello.rb
+
+parts:
+  ruby-hello:
+    plugin: ruby
+    source: .
+    stage:
+      - bin
+      - lib
+    prime:
+      - bin
+      - lib
+    install: |
+      cp ./ruby-hello.rb $SNAPCRAFT_PART_INSTALL/ruby-hello.rb
```

Comment:
This looks rather awkward. It's the plugin's job to pick up files and put them in sensible places by default... isn't there a proper way to deploy ruby scripts?

Reply:
Yeah, make it a gem. Without that, no, it's like Python. No setup.py? Copy .py files around.